bake/bake.go

```diff
@@ -558,7 +558,7 @@ func (c Config) newOverrides(v []string) (map[string]map[string]Override, error)
 			// IMPORTANT: if you add more fields here, do not forget to update
 			// docs/bake-reference.md and https://docs.docker.com/build/bake/overrides/
 			switch keys[1] {
-			case "output", "cache-to", "cache-from", "tags", "platform", "secrets", "ssh", "attest", "entitlements", "network":
+			case "output", "cache-to", "cache-from", "tags", "platform", "secrets", "ssh", "attest", "entitlements", "network", "annotations":
```

Comment:
Should we do the same for `labels`?

Reply:
For what it's worth, I don't think labels has the same issue. It uses `Value` from the overrides: https://github.com/docker/buildx/blob/52f503e8068d4a09e7c1b17e4b88ae75b1f64651/bake/bake.go#L891-L898

Compare to `annotations`: https://github.com/docker/buildx/blob/52f503e8068d4a09e7c1b17e4b88ae75b1f64651/bake/bake.go#L973-L974

So this might already work for `labels`? It looks like it's documented in there to work.

Reply:
Just to clarify, this `switch` is a list of "use `ArrValue` to fill in the overrides" and the default is to use `Value` if you're not in the list.

Reply:
Ah yes indeed, thanks